ddl/backfilling.go

```diff
@@ -850,3 +935,36 @@ func logSlowOperations(elapsed time.Duration, slowMsg string, threshold uint32)
 		logutil.BgLogger().Info("[ddl] slow operations", zap.Duration("takeTimes", elapsed), zap.String("msg", slowMsg))
 	}
 }
+
+// doneTaskKeeper keeps the done tasks and update the latest next key.
+type doneTaskKeeper struct {
+	doneTaskNextKey map[int]kv.Key
```

Comment:
Maybe use a slice is better. But not required

Reply:
Then we need to iterate the slice every time. It is less efficient.